community/kernel/src/main/java/org/neo4j/kernel/impl/storageengine/impl/recordstorage/RecordStorageEngine.java

```diff
@@ -399,6 +404,17 @@ public void init() throws Throwable
         labelScanStore.init();
     }
 
+    public void initFromSnapshot(CountsSnapshot snapshot)
```

Comment:
i do not really think that RecordStorageEngine is a good place for those 2 methods